### PR TITLE
Add singularity env tests

### LINKED ISSUE #2001: 2.x->3.0 Test (25-environment.sh)
I have a branch of this tested against 2.6 that is working.

#1994 currently blocking the pull request. I can create the pull request and put in a test skip if everyone thinks that's a better idea.

### PR DESCRIPTION
**Pull request by jscook2345**
**Description of the Pull Request (PR):**

Porting of 25-environment.sh test to new Go framework

**This fixes or addresses the following GitHub issues:**

Closes #2001 

**Checkoff for all PRs:**

- [ ] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [X] I have tested this PR locally with a `make test`
- [X] This PR is against the project's master branch
- [ ] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [X] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: src/cmd/singularity/env_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/singularityware/singularity/src/pkg/test"
+)
+
+func TestSingularityEnv(t *testing.T) {
+	if os.Getenv("TEST_26") == "" {
```
Comment (bauerm97) [inline src/cmd/singularity/env_test.go]: Should rather use the accepted way to skip disabled tests

```
	if !*runDisabled {
		t.Skip("disabled until issue addressed") // TODO
	}
```

**Review by bauerm97 — DISMISSED** [review]

File: src/cmd/singularity/env_test.go (hunk 1 above)
Comment (bauerm97) [reply src/cmd/singularity/env_test.go]: `go test --run_disabled ...` flag is defined in `src/cmd/singularity/cli_test.go`

File: src/cmd/singularity/env_test.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/singularityware/singularity/src/pkg/test"
```
Comment (ArangoGutierrez) [inline src/cmd/singularity/env_test.go]: update to sylabs

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

**Review by ArangoGutierrez — APPROVED** [review]

**Comment by ArangoGutierrez** [comment]
@bauerm97 <- b46bce0
RFR

**Review by ikaneshiro — APPROVED** [review]

**Comment by jscook2345** [comment]
I'm ready for this to be merged. If #2045 is merged first, I can test that than rebase to merge this.

**Comment by bauerm97** [comment]
@jscook2345 Just merged #2045, go ahead and do that rebase and then it's good to go

**Pull request merged**